libs/auth/src/callback.tsx

```diff
@@ -1,8 +1,9 @@
+import React from "react";
 import { GetServerSideProps } from "next";
-import { H1 } from "components/Typography";
+import { H1 } from "@artsy-studio/components";
```

Comment:
Why not keep this at the Palette level?

Reply:
Ha! Probably should be. Related to our conversation about headline styles.

It will be removed later, just a simplification for now.